os/arch/arm/src/armv7-r/Kconfig

```diff
@@ -7,7 +7,7 @@ comment "ARMv7-R Configuration Options"
 
 config ARMV7R_HAVE_GICv2
 	bool "ARMV7R_GICv2 support"
-	default y
+	default y  if !ARCH_CHIP_BCM4390X
```

Comment:
Do you want to disable GICv2 on BCM chip or to remove default value? This change means later one. Is it right?

Reply:
If default value will be changed into "n", it's good to me.  how do you want to treat it? 

BR, HJ Lee.